addons/website_blog/views/snippets/s_blog_posts.xml

```diff
@@ -69,7 +69,11 @@
 <!-- Multi record "Load-time" templates (rendered in JS on page load) -->
 <!-- List layout -->
 <template id="dynamic_filter_template_blog_post_list" name="List">
-    <div t-foreach="records" t-as="data" class="s_blog_posts_post border-0 rounded-3" data-extra-classes="g-4" data-column-classes="col-12 col-sm-6 col-lg-4">
+    <!-- Filter information used by /website/snippet/filter_templates:
```

Comment:
Maybe add something explaining that the comment is really important, should not be removed and that it is processed in python so the information is clear (since it is "weird" for a comment to be important, let's make sure that it is clear) :slightly_smiling_face: